sihl/src/database_migration.ml

```diff
@@ -262,12 +321,20 @@ struct
   ;;
 
   let start () =
-    if Core_configuration.is_test ()
-    then Lwt.return ()
-    else check_migrations_status ()
+    let open Lwt.Syntax in
+    Core_configuration.require schema;
+    let* () =
+      if Core_configuration.is_test ()
+      then Lwt.return ()
+      else check_migrations_status ()
+    in
+    setup ()
   ;;
 
-  let stop () = Lwt.return ()
+  let stop () =
+    registered_migrations := Map.empty;
```

Comment:
Is this correct? So if we restart this service (without restarting everything else), all migrations have to be re-registered?

Reply:
Good catch, it's not correct :) Fixed.